### PR TITLE
Modified copyright in license header (resolves #989)

### LINKED ISSUE #989: Copyright header should refer to `Regents of the University of California`
… because we have multiple UCs contributingt to Toil.

See also https://github.com/BD2KGenomics/toil-scripts/pull/311

### PR DESCRIPTION
**Pull request by fnothaft**

### CODE REVIEW

**Comment by hannes-ucsc** [comment]
Ran out of domains/buckets.

Jenkins, test this please.

File: version.py [hunk 1]
```diff
@@ -1 +0,0 @@
-src/toil/version.py
```
Comment (hannes-ucsc) [inline version.py]: This must be a symlink. Looks like you replaced the symlink with a copy?

Comment (fnothaft) [inline version.py]: Yeah, not sure how that happened. Let me patch this up tomorrow.

Comment (fnothaft) [inline version.py]: I think a similar thing may have happened on https://github.com/BD2KGenomics/toil-scripts/pull/311 as well.

**Comment by hannes-ucsc** [comment]
Continues in #999

**Pull request closed without merging**